lib/wallaroo/core/source/tcp_source/tcp_source_coordinator.pony

```diff
@@ -157,7 +157,7 @@ actor TCPSourceCoordinator[In: Any val] is SourceCoordinator
     end
 
     @printf[I32]((pipeline_name + " source will listen (but not yet) on "
-          + host + ":" + service + "\n").cstring())
+          + host + ":" + service + " max size " + max_size.string() + "\n").cstring())
```

Comment:
I'd add `TCP Source max buffer size:` to be a bit more descriptive

Reply:
Good plan. The new logging looks like this:
```
1603759261.068373,Onnx Models source will listen (but not yet) on 0.0.0.0:7102
1603759261.070539,Onnx Models TCP Source max buffer size: 200000000
```